src/ts/ConvivaAnalytics.ts

```diff
@@ -556,7 +556,8 @@ export class ConvivaAnalytics {
     }
 
     this.onPlaybackStateChanged(event);
-    this.internalEndSession(event);
+    this.convivaVideoAnalytics.release();
```

Comment:
This was preventing a new session from starting. This was a bug I created that did not exist in 2.x but its fixed now